.github/workflows/update_grafana_dashboards.yml

```diff
@@ -0,0 +1,33 @@
+name: Update Grafana Dashboards
+
+on:
+  schedule:
+    - cron: "*/15 * * * *"
+
+jobs:
+  update-dashboards:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install requirements
+        run: |
+          set -eux
+          pip install requests==2.24.0
+      - name: Update Dashboards
+        id: update
+        env:
+          GRAFANA_TOKEN: ${{ secrets.GRAFANA_TOKEN }}
```

Comment:
is the GRAFANA_TOKEN provisioned manually or included in the provision script? If it's manually created, the token might get lost when the whole grafana is reset

Reply:
Ah good catch, yes it is manually provisioned via the UI and will break on updates. Apparently we can use HTTP basic auth instead so that should work fine as a replacement https://github.com/grafana/grafana/issues/2431#issuecomment-127287390